src/plotly_plots.py

```diff
@@ -9,8 +9,17 @@
 
 from src import client_types, int_bucket_label
 
+font_family = "Work Sans"
```

Comment:
```suggestion
pio.templates[pio.templates.default].layout.font.family = "Work Sans"
pio.templates[pio.templates.default].layout.hoverlabel.font.family = "Work Sans"
```

Überschreib einfach die Standardschriftarten. Das ist alles was du brauchst. Musste dann nicht in jedem Plot herumoperieren.